cmd/metricswatcher/metrics.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/mattermost/mattermost-load-test-ng/cmd/metricswatcher/prometheushelper"
+
+	"github.com/mattermost/mattermost-server/v5/mlog"
+)
+
+type PrometheusQuery struct {
+	Description string
+	Query       string
+	Threshold   float64
+	Alert       bool
+}
+
+func checkMetrics(queryFile string) {
+	var (
+		prometheusQueries = readPrometheusQueriesFile(queryFile)
+		prometheusHelper  = createPrometheusHelper()
+	)
+
+	for {
+		checkQueries(prometheusHelper, prometheusQueries)
+
+		time.Sleep(time.Duration(configuration.PrometheusConfiguration.UpdateIntervalInMS) * time.Millisecond)
```

Comment:
how should this loop terminate?

Reply:
It won't. It keeps running until you Ctrl-C the application.